index.html

```diff
@@ -476,6 +311,188 @@
   </div>
   <!-- ends Transcode to H.265 -->
 
+  <p>&nbsp;</p>
```

Comment:
To make an optical gap between video and audio-only commands.
First wanted to introduce an "audio" headline, but that looked bloated...